SpatialGDK/Source/SpatialGDK/Public/SpatialCommonTypes.h

```diff
@@ -6,6 +6,15 @@
 
 #include <WorkerSDK/improbable/c_worker.h>
 
+// IMPORTANT: This is required for Linux builds to succeed - don't remove!
+// Worker_EntityId from the Worker SDK resolves to a long on Linux.
+// These are not a type of key supported by TMap.
+using Worker_EntityId_Key = int64;
```

Comment:
I believe you'll need to add a similar definition for the component id for linux compilation. You can test building linux locally within VS if you have the toolchain setup.

Reply:
(discussed offline, no need to do this as not using component ID as a key anywhere that;s not a UProperty where we need to explicitly write the primitive type anyway)